config/shenzen_neo/nas-wr01ze.xml

```diff
@@ -0,0 +1,85 @@
+      <Help>
+        This parameter defines the LED indication Function ON/OFF.
+        If disabled, then LED indication Function and will always be turn-off, while enabled means LED Function is on.
+      </Help>
+      <Item label="Disable" value="0"/>
+      <Item label="Enable" value="1"/>
+    </Value>
+	<Value type="list" genre="config" instance="1" index="4" label="Configure Plug Time switch Function" value="1" size="1">
+      <Help>
+        This parameter defines the timer function Enable/Disable.
+        If disabled, then time switch function is not available.
+        The time period will be defined in parameter #5.
+        If enabled, and when turn the plug relay on, the timer in plug start run with time period defined in parameter #5 and the plug will turn the relay off.
+      </Help>
+      <Item label="Disable" value="0"/>
+      <Item label="Enable" value="1"/>
+    </Value>
+	<Value type="short" genre="config" instance="1" index="5" label="Configure Time switch Period" units="minute" value="150" min="1" max="32767" size="2">
+      <Help>
+        This parameter defines the time period that plug time switch off. This parameter can be configured 1 - 32767(in minutes).
+        If Parameter #4 is enabled, and relay is turn-on, the relay will be turn-off after delay this parameter.
+      </Help>
+    </Value>
+    <Value type="list" genre="config" instance="1" index="6" label="Send Meter Report" value="1" size="1">
+      <Help>This parameter defines Disable/Enable meter report measure data to controller.</Help>
+      <Item label="Disable" value="0"/>
+      <Item label="Enable" value="1"/>
+    </Value>
+    <Value type="short" genre="config" instance="1" index="7" label="Meter Report Interval" units="second" value="300" min="30" max="32767" size="2">
+      <Help>This parameter defines interval time (30-32767 seconds) that Meter report data to main controller .</Help>
+    </Value>
+    <Value type="byte" genre="config" instance="1" index="8" label="Configure maximum over-load current" units="Ampere" value="13" min="1" max="16" size="1">
+      <Help>
+        This parameter defines maximum current (in Amperes) the plug can provide to load that be connected to plug.
+        If the current consumed by load is greater than maximum current, the plug will cut off power, and send NOTIFICATION_EVENT_POWER_MANAGEMENT_OVER_LOAD_DETECTED to associated devices in lifeline. Red Led will blink every seconds until current is normal (less than Parameter #8).
+        You must remove the loader and turn on the Plug (press the button once) to enable current monitor and clear NOTIFICATION ALARM. This parameter value must be great than #10.
+        If user set this parameter to default by Configure CC, the parameter #10 will be set to default value.
+      </Help>
+    </Value>
+	 <Value type="byte" genre="config" instance="1" index="9" label="Configure power report" units="%" value="30" min="1" max="100" size="1">
```

Comment:
Per [this](https://community.smartthings.com/t/release-neo-coolcam-power-plug/144274/108), this param may now be in `cA`  (centiamp), with a valid range of `1-1600`.
Not sure if the source of the info is reliable (a manual) or if it comes from trial and error, though it would match my own observations [here](https://community.openhab.org/t/neo-coolcam-z-wave-devices/18868/307?u=mbronk)

I.e. I suggest correcting the unit and the range.

Reply:
@mbronk Good #9 suggestion, i just updated the file #9 with your suggestion and changed some default values.
The sources i used to create the xml file is a combination of online references (the shipped device manual is incorrect) and trial and error.
All parameters are working (tested myself) except parameter 9 (of course this is not optimal and needs a change in the future)